stable/vulcan/templates/api/deployment.yaml

```diff
@@ -76,6 +76,10 @@ spec:
             value: {{ .Values.comp.conf.persistenceHost | default  (include "persistence.fullname" .) | quote }}
           - name: VULNERABILITYDB_URL
             value: {{ .Values.comp.conf.vulndbapiUrl | default  (include "vulndbapi.url" .) | quote }}
+          - name: VULCANTRACKER_URL
```

Comment:
What is the behaviour of the new Vulcan api in case that the tracker is disabled? Empty value for the.tracker url?

I would protect those envs with an `if .Values.traker.enabled `